airbyte-integrations/connectors/source-datadog/source_datadog/spec.yaml

```diff
@@ -0,0 +1,56 @@
+documentationUrl: https://docs.airbyte.com/integrations/sources/datadog
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Datadog Source Spec
+  type: object
+  required:
+    - api_key
+    - application_key
+  additionalProperties: true
+  properties:
+    api_key:
+      title: API Key
+      description: Datadog API key
+      type: string
+      airbyte_secret: true
+      order: 1
+    application_key:
+      title: Application Key
+      description: Datadog application key
+      type: string
+      airbyte_secret: true
+      order: 2
+    query:
+      title: Query
+      description: The search query. This just applies from Logs and Audit Logs entities.
+      type: string
+      order: 3
+    limit:
+      type: integer
+      title: Limit
+      default: 5000
+      minimum: 1
+      maximum: 5000
+      description: Maximum number of records to collect per request.
+      order: 4
```

Comment:
I'd like to call this record_limit or max_record_per_request to make the variable name clear.

Reply:
Done!